.github/workflows/release.yaml

```diff
@@ -22,3 +22,24 @@ jobs:
         Expire-Date: 0
         EOF
         DRY_RUN=true BRANCH=main ./scripts/release.sh --no-upload --no-docker-push 3.6.99
+  release35:
```

Comment:
My immediate feeling is that probably we should add all release-3.5 related changes into branch `release-3.5` only. It's clearer to me, but the bad side is we need maintain different stuff/scripts for different releases (usually 3).

Maintaining everything only in the main branch is also an option. The good point is we don't need to maintain multiple copies (with only minor differences) ; but the bad side is we complicate the release script & config.  

It's a tradeoff. I prefer to the first option (maintain different scripts/configs for different releases).

Reply:
Agree. If the testing scenario pre-existed in the the 3.5 branch... we would just update it when creating the 3.6 branch (main) with 3.6 specific changes.